pkg/tls/renewer.go

```diff
@@ -95,7 +97,7 @@ func (c *certRenewer) RenewCA(ctx context.Context) error {
 	}
 	now := time.Now()
 	certs = removeExpiredCertificates(now, certs...)
-	if !allCertificatesExpired(now.Add(5*c.certRenewalInterval), certs...) {
```

Comment:
Why not update `CertRenewalInterval` (pkg/tls/renewer.go) directly?

Reply:
AFAIK this is the validity check performed every 12 hours and it is used in several places. To avoid any confusion, I have created a new constant called `RenewBefore` which clearly indicates its purpose. Instead, I can use `30 * c.certRenewalInterval`

Reply:
Makes sense!